#[161348790] [Document API using Swagger]

brucemakallan opened the pull request:
#### What does this PR do?
Use "Django REST Swagger" to document the API

#### How should this be manually tested?
1. `pip install django-rest-swagger`
3. Run the app: `python manage.py runserver`
The API documentation will be available on the landing page.

#### What are the relevant pivotal tracker stories?
[#161348790] Document API using swagger

Sulaiman-Mozes committed:
#161348782 Upgrade Authors Haven Django app from Django 1.10 to Django 2.1 (#1)

* chore(requirements.txt) :
- upgraded from django 1.10 to 2.1
[Fixes #161348782]

* change Django URL routing syntax from url to path [Finishes #161348782]

Commit:
chore (Document with Swagger):

- Use "Django REST Swagger" to document the API

[Finishes #161348790]

emugaya commented on .gitignore:
```diff
@@ -91,3 +91,6 @@ ENV/
 db.sqlite3
+
+#VSCode hidden folder
+.vscode/
```
Add newline to end of this file @brucemakallan

emugaya commented on authors/settings.py:
```diff
@@ -43,6 +43,8 @@
     'authors.apps.authentication',
     'authors.apps.core',
     'authors.apps.profiles',
+
```
Remove line

emugaya commented on authors/settings.py:
```diff
@@ -138,6 +140,6 @@
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
+        #'authors.apps.authentication.backends.JWTAuthentication',
```
Why are you commenting this out? @brucemakallan

brucemakallan replied:
It was commented out in the PR from Sulaiman. I ended up with two commits but I've since brushed up on my version control skills. He commented it out because JWT is not yet implemented and that line produces lots of errors because it expects the implementation to be present.

emugaya replied:
ok

emugaya reviewed (changes requested):
Address my comments

Commit:
fix chore (Document with Swagger):

- Remove unnecessary spacing
- Add spacing at the end of gitignore file

[Fixes #161348790]
